internal/expr/parser.go

```diff
@@ -89,14 +97,19 @@ func (pe *ParsedExpr) String() string {
 	return out.String()
 }
 
+// bypass adds the string between the previous I/O part and the current part.
+func (p *Parser) addBypass() {
+	if p.prevPart != p.partStart {
+		p.parts = append(p.parts, &bypassPart{p.input[p.prevPart:p.partStart]})
+	}
+}
+
 // add pushes the parsed part to the parsedExprBuilder along with the bypassPart
 // that stretches from the end of the previous part to the beginning of this
 // part.
 func (p *Parser) add(part queryPart) {
-	// Add the string between the previous I/O part and the current part.
 	if p.prevPart != p.partStart {
-		p.parts = append(p.parts,
-			&bypassPart{p.input[p.prevPart:p.partStart]})
+		p.parts = append(p.parts, &bypassPart{p.input[p.prevPart:p.partStart]})
 	}
 
 	if part != nil {
```

Comment:
Just from our discussion, could this be removed due to the new `addBypass` method?

Reply:
See comment above.